Gemfile.lock

```diff
@@ -21,7 +22,8 @@ PLATFORMS
   ruby
 
 DEPENDENCIES
+  relaxed-rubocop
   rubocop (~> 0.76.0)
 
 BUNDLED WITH
-   1.9
+   1.17.3
```

Comment:
I am a bit worried about this, as I've been getting errors about mismatched bundler version. I wonder how we can work around it.